acceptance/script.prepare

```diff
@@ -47,3 +47,14 @@ title() {
     local label="$1"
     printf "\n=== %s" "$label"
 }
+
+withdir() {
+    local dir="$1"
+    shift
+    local orig_dir="$(pwd)"
+    cd "$dir" || return $?
+    "$@"
+    local exit_code=$?
+    cd "$orig_dir" || return $?
+    return $exit_code
+}
```

Comment:
It's easier/shorter to just use a subshell.

E.g.
```
(
  cd subdir/a/b
  errcode trace $CLI bundle validate -o json | jq .bundle.git
)
```

Reply:
So we're comparing:

script:
`errcode trace withdir subdir/a/b $CLI bundle validate -o json | jq .bundle.git`

output:
`>>> withdir subdir/a/b $CLI bundle validate -o json`


vs

script:
```
(
    trace cd subdir/a/b
    errcode trace $CLI bundle validate -o json | jq .bundle.git
)
```

output:
```
>>> cd subdir/a/b
>>> $CLI bundle validate -o json
```

Note, in the output you cannot see that you're inside a subshell and that `cd subdir/a/b` only applies to next statemnent.